go/vt/mysqlctl/backup.go

```diff
@@ -225,29 +226,32 @@ func Restore(
 	localMetadata map[string]string,
 	logger logutil.Logger,
 	deleteBeforeRestore bool,
-	dbName string) (mysql.Position, error) {
+	dbName string,
+	snapshotTime time.Time) (mysql.Position, string, error) {
 
-	rval := mysql.Position{}
+	var rval mysql.Position
+	zeroPosition := mysql.Position{}
```

Comment:
We could do that until we get to the call to ExecuteRestore. After that, even if rval is non-zero, under various error conditions we return zeroPosition. It seemed clearer to just use zeroPosition everywhere rather than use rval up to that call and zeroPosition afterwards.

Reply:
If we need to make sure it's zero, I feel like the form `mysql.Position{}` is more clear for readers. You immediately know exactly what type it is and that it's the zero value, instead of having to go check what the variable `zeroPosition` is. That's just my style opinion though, so feel free to keep it if you disagree.

Reply:
I had actually changed `mysql.Position{}` to `zeroPosition` in a couple of other files based on feedback on a previous PR. I changed it here to make it consistent with that, so I would rather leave it as is.

Reply:
Alright, sounds like another reviewer has a different style opinion. I'll drop it then.